fedoragsearch-transforms/islandora_transforms/slurp_all_MODS_to_solr.xslt

```diff
@@ -37,32 +37,74 @@
   </xsl:template>
 
   <!-- utk_ir_MODS mode -->
-  <!-- the following template creates an _ms field for author(s) -->
-  <xsl:template match="mods:mods/mods:name[(mods:role/mods:roleTerm='Author') or (mods:role/mods:roleTerm='author')]" mode="utk_ir_MODS">
+
+
+  <!-- JIRA TRAC-875 Define utk_mods_etd_name_author_ms in Solr -->
```

Comment:
More author stuff.

Reply:
I think at this point, the simplest thing for me to do is to start over.  I have the template code for the committee-member.  I can close this pull request, make a new branch, add my template code and  nothing else and resubmit a pull request.  Where should I start the new branch from?
institutional_repository or master?
@markpbaggett @CanOfBees